docs/root/configuration/http/http_conn_man/stats.rst

```diff
@@ -3,121 +3,121 @@
+   ``downstream_rq_failed_path_normalization``, Counter, Total requests redirected due to different original and normalized URL paths or when path normalization failed. This action is configured by setting the :ref:`path_with_escaped_slashes_action <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.path_with_escaped_slashes_action>` config option.
+   ``downstream_rq_1xx``, Counter, Total 1xx responses
+   ``downstream_rq_2xx``, Counter, Total 2xx responses
+   ``downstream_rq_3xx``, Counter, Total 3xx responses
+   ``downstream_rq_4xx``, Counter, Total 4xx responses
+   ``downstream_rq_5xx``, Counter, Total 5xx responses
+   ``downstream_rq_ws_on_non_ws_route``, Counter, Total upgrade requests rejected by non upgrade routes. This now applies both to WebSocket and non-WebSocket upgrades
+   ``downstream_rq_time``, Histogram, Total time for request and response (milliseconds)
+   ``downstream_rq_idle_timeout``, Counter, Total requests closed due to idle timeout
+   ``downstream_rq_max_duration_reached``, Counter, Total requests closed due to max duration reached
+   ``downstream_rq_timeout``, Counter, Total requests closed due to a timeout on the request path
+   ``downstream_rq_overload_close``, Counter, Total requests closed due to Envoy overload
+   ``downstream_rq_redirected_with_normalized_path``, Counter, Total requests redirected due to different original and normalized URL paths. This action is configured by setting the :ref:`path_with_escaped_slashes_action <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.path_with_escaped_slashes_action>` config option.
+   ``rs_too_large``, Counter, Total response errors due to buffering an overly large body
 
 Per user agent statistics
 -------------------------
 
-Additional per user agent statistics are rooted at *http.<stat_prefix>.user_agent.<user_agent>.*
-Currently Envoy matches user agent for both iOS (*ios*) and Android (*android*) and produces
+Additional per user agent statistics are rooted at ``http.<stat_prefix>.user_agent.<user_agent>.``
+Currently Envoy matches user agent for both iOS (``ios``) and Android (``android``) and produces
 the following statistics:
 
 .. csv-table::
    :header: Name, Type, Description
    :widths: 1, 1, 2
 
-   downstream_cx_total, Counter, Total connections
-   downstream_cx_destroy_remote_active_rq, Counter, Total connections destroyed remotely with 1+ active requests
-   downstream_rq_total, Counter, Total requests
+   ``downstream_cx_total``, Counter, Total connections
+   ``downstream_cx_destroy_remote_active_rq``, Counter, Total connections destroyed remotely with 1+ active requests
+   ``downstream_rq_total``, Counter, Total requests
 
 .. _config_http_conn_man_stats_per_listener:
 
 Per listener statistics
 -----------------------
 
-Per listener statistics are rooted at *listener.<address>*.
+Per listener statistics are rooted at ``listener.<address>``.
```

Comment:
nit:
```suggestion
Per listener statistics are rooted at ``listener.<address>.``
```

Reply:
i think it needs both - the `.` in the stats label and a full stop

Reply:
FWIW, I've seen this in other places around the docs (e.g., lines 133, 135 of this file).
Not sure if it is worth updating all the places, but if we can have some clear guideline, we can gradually move the docs there.